dev/tracing_tests/README.md

```diff
@@ -0,0 +1,7 @@
+# Tracing tests
+
+The tests in this folder must be run with `flutter test --enable-vmservice`,
```

Comment:
Any chance you can check for this condition from within the test and just print this message if the test was run incorrectly?

Reply:
(Readmes usually get ignored..)

Reply:
Never mind, I see below that you already do this: 🌟